tests/static-api/teams/wg-test.toml

```diff
@@ -1,5 +1,6 @@
 name = "wg-test"
 kind = "working-group"
+top-level = true
```

Comment:
I think we should also validate that working groups cannot be top-level teams.

Reply:
Sure! I have added a check.